caiman/base/movies.py

```diff
@@ -1593,7 +1587,7 @@ def load(file_name: Union[str, list[str]],
 
         elif extension == '.mmap':
             filename = os.path.split(file_name)[-1]
-            Yr, dims, T = load_memmap(
+            Yr, dims, T = caiman.mmapping.load_memmap(
```

Comment:
For reasons I can't explain very well, I like this expanded version of this as it says this is a caiman memory mapping mechanism we are getting into (people might think it is numpy or something or get confused). There is no hard fact of the matter here about the right cutoff I'm just throwing my intuitions here and doing my best to explain them 😬